common-template/src/main/resources/templates/google_chat/OCM/ocmDefault.json

```diff
@@ -0,0 +1 @@
+{"text":"{data.events.size()} event{#if data.events.size() > 1}s{/if} triggered from {data.source.application.display_name} - {data.source.bundle.display_name}. <{#if data.context.environment_url??}{data.context.environment_url}/{data.ocm_cluster_path}?from=notifications&integration=google_chat{#else}{data.application_url}{/if}|Open {data.source.application.display_name}>"}
```

Comment:
Because those templates are dedicated to OCM and `subscription_id` is always present in their messages,  we should keep it simple as email logic like:
`{"text":"{data.events.size()} event{#if data.events.size() > 1}s{/if} triggered from {data.source.application.display_name} - {data.source.bundle.display_name}. <https://cloud.redhat.com/openshift/details/s/{action.events[0].payload.global_vars.subscription_id}#overview?from=notifications&integration=google_chat|Open {data.source.application.display_name}>"}`

We may add complexity later regarding next enhancement to add more details on those notifications.
(same comment for Teams and Slack)

Reply:
I can make that change, I was just concerned that if the format is ever changed, the alerts wouldn't send. But if it's stable, I'll make this change.